<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- <param name = "camera_in_topic" type="string" value="/argus/ar0234_front_right/image_raw"/>
  <param name = "lidar_in_topic" type="string" value="/$(env ROBOT_NAMESPACE)/velodyne_points"/> -->

  <param name = "topic_input_image_and_cloud" type="string" value="/$(env ROBOT_NAMESPACE)/image_and_cloud"/>
  <param name = "topic_input_images_and_cloud" type="string" value="/$(env ROBOT_NAMESPACE)/images_and_cloud"/>
  <param name = "topic_output_velodyne_cloud" type="string" value="/$(env ROBOT_NAMESPACE)/velodyne_cloud"/>
  <param name = "topic_output_l515_cloud" type="string" value="/$(env ROBOT_NAMESPACE)/l515_cloud"/>
  <param name = "topic_output_projected_image" type="string" value="/$(env ROBOT_NAMESPACE)/projected_image"/>

  <param name = "camera_name" type="string" value="camera"/>
  
  <param name = "cam_config_file_path" type="string" value="$(find camera_calib)/config/dog2_argus_intrinsic.yaml" />
  <param name = "result_file" type="string" value="$(find camera_calib)/results/dog2_argus_fr_l515_C_T_L.txt" />

  <param name= "project_only_plane" type="bool" value="false"/>

  <param name = "dist_cut_off" type="int" value="5"/>

  <node  pkg="camera_calib" name="cam_lidar_proj" type="cam_lidar_proj_node" output="screen" />

  <rosparam file="$(find camera_calib)/config/projection_params.yaml" />

  <!-- <node pkg="tf" type="static_transform_publisher" name="lidar_broadcaster" args="0 0 0 0 0 0 1 map multijackal_02 100" /> -->
  
  
</launch>
